.github/CONTRIBUTING.md

```diff
@@ -1,11 +1,13 @@
-# ReDoc Contributing Guide
+# Redoc Contributing Guide
 
-Hi! We're really excited that you are interested in contributing to ReDoc. Before submitting your contribution though, please make sure to take a moment and read through the following guidelines.
+Hi! We're really excited that you are interested in contributing to Redoc. Before submitting your contribution though, please make sure to take a moment and read through the following guidelines.
 
-- [Issue Reporting Guidelines](#issue-reporting-guidelines)
-- [Pull Request Guidelines](#pull-request-guidelines)
-- [Development Setup](#development-setup)
-- [Project Structure](#project-structure)
+- [Redoc Contributing Guide](#redoc-contributing-guide)
+  - [Issue Reporting Guidelines](#issue-reporting-guidelines)
+  - [Pull Request Guidelines](#pull-request-guidelines)
+  - [Development Setup](#development-setup)
+    - [Commonly used NPM scripts](#commonly-used-npm-scripts)
+  - [Project Structure](#project-structure)
 
 ## Issue Reporting Guidelines
```

Comment:
I notice everywhere else in our docs and in our style guide we use sentence-case headings. Should this be adjusted here?